app/controllers/depositions_controller.rb

```diff
@@ -0,0 +1,51 @@
+class DepositionsController < ApplicationController
+
+  before_filter :authenticate_user!
+
+  def new
+    @deposition = Deposition.new(submission: submission, user: current_user)
+  end
+
+  def create
+    @deposition = Deposition.new(
+      { submission: submission, user: current_user }.merge(deposition_create_params)
```

Comment:
Why not merge `submission` and `current_user` inside `deposition_create_params` method?